content/participants/ttjc@Manoj-M-S/index.md

```diff
@@ -64,7 +64,7 @@ It would be good if someone can guide me in frontend, backend and database secti
    Technologies used:
    *Frontend : React JS
    *Backend : Node JS, Express
-   \*Database: Mongodb
+   *Database: Mongodb
 
 [Github](https://github.com/Manoj-M-S/Clone-Client)
```

Comment:
you need to put proper spacing and align them, put two spaces after each line, or add a new line the same goes throughout the file.